runtime/pendulum/src/zenlink.rs

```diff
@@ -6,9 +6,9 @@ use orml_traits::MultiCurrency;
 use sp_runtime::{DispatchError, DispatchResult};
 use sp_std::marker::PhantomData;
 
+use spacewalk_primitives::CurrencyId;
```

Comment:
Also here, if we export the `CurrencyId` without renaming from `lib.rs`, I think we wouldn't even need this `use` statement here because it should be imported by `use super::*` already (?). Should be double-checked

Reply:
I've checked this, and it has worked well.